wpiutil/src/main/java/edu/wpi/first/util/CircularBuffer.java

```diff
@@ -19,11 +21,9 @@ public class CircularBuffer {
    *
    * @param size The size of the circular buffer.
    */
+  @SuppressWarnings("unchecked")
   public CircularBuffer(int size) {
-    m_data = new double[size];
-    for (int i = 0; i < m_data.length; i++) {
-      m_data[i] = 0.0;
-    }
```

Comment:
I think we should explicitly initialize to null (and also explicitly initialize the other fields), and/or call `clear()`.